README.md

```diff
@@ -58,7 +58,13 @@ $ npm start
 
 Open your web browser and navigate to `http://localhost:3000/`.
 
-(See also: [recipe for running blip locally with hot module replacement](http://developer.tidepool.io/docs/recipes/index.html#a-running-the-platform-locally-with-runservers-but-blip-with-hot-module-replacement-hmr-via-webpack 'Tidepool developer portal: front end recipes').)
+(See also: [recipe for running blip locally with hot module replacement](http://developer.tidepool.io/docs/front-end/recipes.html#a-running-the-platform-locally-with-runservers-but-blip-with-hot-module-replacement-hmr-via-webpack 'Tidepool developer portal: front end recipes').)
+
+### Redux dev tools
+
+Blip includes several Redux developer tools: the original time-travel dev tools UI, a console action logger, and a mutation tracker for catching mutations to the state tree (which should be immutable). The last of these in particular is performance killer (though *none* of them could even be said to have a minimal effect on performance). By default when running for local development with `npm start` (which means `NODE_ENV` is `development`), the `DEV_TOOLS` flag will be `true`, and all of these dev tools will be active. Because they affect performance profoundly, this may not always be desirable. To turn *off* the dev tools in development, kill the Webpack HMR build, run `export DEV_TOOLS=false`, then start up blip again with `npm start`.
```

Comment:
is _a_ performance killer.

Is "kill the Webpack HMR" synonymous with stopping `npm start`? There's no other context in the README for what "Webpack HMR" means, so it would be nice to other add context in brackets after that, or mention it elsewhere on the page.